src/types.rs

```diff
@@ -1388,3 +1388,51 @@ define_bounded_integers! {
     BoundedI64 i64 signed -> i64,
     BoundedIsize isize signed -> isize,
 }
+
+//Indexing operations on [T; N], Vec<T> and VecDeque<T> for BoundedUsize
+#[cfg(all(feature = "std", feature = "types"))]
```

Comment:
The `feature = "std"` part can be removed since arrays (and slices, as below) are available even when `std` isn’t (as they’re part of the core language). Additionally, the `feature = "types"` part should be unnecessary since the entire module is gated by that feature, so you can just remove this line. That second part applies to all the other instances as well.